src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java

```diff
@@ -50,9 +51,17 @@
     private static final Logger LOGGER = Logger.getLogger(WorkflowMultiBranchProject.class.getName());
 
     static final String SCRIPT = "Jenkinsfile";
+    private static final String SCRIPT_LOWERCASE = SCRIPT.toLowerCase(Locale.ENGLISH);
     static final SCMSourceCriteria CRITERIA = new SCMSourceCriteria() {
         @Override public boolean isHead(SCMSourceCriteria.Probe probe, TaskListener listener) throws IOException {
-            return probe.exists(SCRIPT);
+            if (probe.exists(SCRIPT)) {
+                return true;
+            } else {
+                if (probe.exists(SCRIPT_LOWERCASE)) {
+                    listener.getLogger().println("Found " + SCRIPT_LOWERCASE + " rather than " + SCRIPT + ", skipping");
```

Comment:
@jglick I prefer to keep the original behavior due to this `listener` breaks the log message structure that we are using in GitHub Branch Source.